internal/http/http.go

```diff
@@ -152,9 +152,21 @@ func SetupRoutes(handle *graph.Handle, cfg *config.Config) {
 
 	//m.Use(martini.Static("static", martini.StaticOptions{Prefix: "/static", SkipLogging: true}))
 	//r.Handler("GET", "/static", http.StripPrefix("/static", http.FileServer(http.Dir("static/"))))
-	r.GET("/docs/:docpage", docs.ServeHTTP)
-	r.GET("/ui/:ui_type", root.ServeHTTP)
-	r.GET("/", root.ServeHTTP)
-	http.Handle("/static/", http.StripPrefix("/static", http.FileServer(http.Dir(fmt.Sprint(assets, "/static/")))))
-	http.Handle("/", r)
+
+	// only show docs when allowed
+	if cfg.HostDocs {
```

Comment:
For the path in the API or the path on disk?
And what about the flag for the UI?